recipes/2dfatmic/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "2dfatmic" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://www.epa.gov/sites/production/files/2014-07/2dfatmic.zip
```

Comment:
```suggestion
  - url: https://www.epa.gov/sites/production/files/{{ version.replace('.', '-') }}/2dfatmic.zip
```
?

Reply:
The version is 1.0, so this change doesn't make sense